### PR TITLE
PMM-3402 Update README.md.

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/proxysql_exporter?pullRequest=24) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/proxysql_exporter?pullRequest=24) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/proxysql_exporter?pullRequest=24) it.</sub>

File: README.md [hunk 1]
```diff
@@ -10,12 +10,23 @@ Supported versions: 1.2 and 1.3.
...
+
+1. Get the code from the Percona repository:
+ 
+    ```bash
+    go get github.com/percona/proxysql_exporter
+    ```
+ 2. Switch to the buld directory and run ``make``:
+ 
+    ```bash
+    cd ${GOPATH-$HOME/go}/src/github.com/percona/proxysql_exporter
```
Comment (AlekSi) [inline README.md]: Oh, that's cool!

File: README.md [hunk 2]
```diff
@@ -10,12 +10,23 @@ Supported versions: 1.2 and 1.3.
...
+    make
```
Comment (AlekSi) [inline README.md]: That will not only build exporter but also run tests, etc. I'm not sure if that what you want.

File: README.md [hunk 3]
```diff
@@ -10,12 +10,23 @@ Supported versions: 1.2 and 1.3.
...
-make
-```
+* [Go compiler](https://golang.org/dl/)
 
+### Building
+
+1. Get the code from the Percona repository:
+ 
+    ```bash
+    go get github.com/percona/proxysql_exporter
```
[the remark this answers is not in the document]
Comment (fiowro) [reply README.md]: Added the -u option. Thanks for the hint!

File: README.md (hunk 2 above)
Comment (fiowro) [reply README.md]: Possibly we could use `make format build` instead, but single `make` looks very familiar, and just spending a moment to run tests does nothing bad...

**Pull request merged**